Source/Charts/Data/Implementations/Standard/PieChartData.swift

```diff
@@ -23,6 +23,20 @@ open class PieChartData: ChartData
         super.init(dataSets: dataSets)
     }
 
+    /// - returns: All DataSet objects this ChartData object holds.
+    @objc open override var dataSets: [IChartDataSet]
+    {
+        get
+        {
+            return super.dataSets
+        }
+        set
+        {
+            assert(newValue.count > 1, "Found multiple data sets while pie chart only allows one")
```

Comment:
Whoops! You want `assert(newValue.count <= 1, "Found multiple data sets while pie chart only allows one")`

Reply:
What was I writing..